src/adapter/sources.ts

```diff
@@ -902,13 +902,25 @@ export class SourceContainer {
 
     try {
       sourceMap.map = await this.sourceMapFactory.load(source.sourceMap.metadata);
-    } catch (e) {
-      this._dap.output({
-        output: sourceMapParseFailed(source.url, e.message).error.format + '\n',
-        category: 'stderr',
-      });
+    } catch (urlError) {
+      try {
+        const sourceMapAbsolutePath = await this.sourcePathResolver.urlToAbsolutePath({
+          url: source.sourceMap.metadata.sourceMapUrl,
+        });
+        if (sourceMapAbsolutePath) {
+          source.sourceMap.metadata.sourceMapUrl = sourceMapAbsolutePath;
```

Comment:
Should this be a file URI for the `sourceMapUrl`?

Reply:
Added call to utils.absolutePathToFileUrl to make it a file URI